telegram/utils/warnings.py

```diff
@@ -29,9 +31,22 @@ class TelegramUserWarning(UserWarning):
     __slots__ = ()
 
 
+class PTBRuntimeWarning(PTBUserWarning, RuntimeWarning):
+    """
+    Custom runtime warning class used for warnings in this library.
+    """
+
+    __slots__ = ()
+
+
 # https://www.python.org/dev/peps/pep-0565/ recommends to use a custom warning class derived from
 # DeprecationWarning. We also subclass from TGUserWarning so users can easily 'switch off' warnings
-class TelegramDeprecationWarning(TelegramUserWarning, DeprecationWarning):
+class PTBDeprecationWarning(PTBUserWarning, DeprecationWarning):
     """Custom warning class for deprecations in this library."""
 
     __slots__ = ()
+
+
+def warn(message: str, category: Type[Warning] = PTBUserWarning, stacklevel: int = 1) -> None:
+    """Helper function used as a shortcut for warning with default values."""
+    warnings.warn(message, category=category, stacklevel=stacklevel)
```

Comment:
maybe

```suggestion
    warnings.warn(message, category=category, stacklevel=stacklevel+1)
```
? Then we wouldn't have to think about that wan calling this function 🤔

Reply:
I guess.. I'll also specify that in the docs